LibreHardwareMonitorLib/Hardware/Control.cs

```diff
@@ -44,9 +58,21 @@ public Control
 
         public ControlMode ControlMode
         {
-            get { return _mode; }
+            get
```

Comment:
Why are two control modes now required? It seems unnecessary to have two. Also all ifs must be separated (same in other files) to match code style. Also brackets need to be added for multi blocks.

Reply:
This was from the original PR, I think they added two because "manual" just specifies a fixed value. 

If the question is "why is ControlMode and ActualControlMode required?" I'm not sure, I'll have to dig into it. I suspect it'd be good to still have "default", "manual" and  "sensor curve" or something like that as control modes. 

I've also noticed that it's not saving out the control curve and you have to re-apply it each time LHM boots, which is less than ideal.

Reply:
This could be done differently for sure, but I was mainly trying to merge this feature from another branch (which I also happened to want to use)

I've changed the UI to better reflect (and have shorter names for) what's going on with this.

I think we should figure what this project wants to do in general with respects to device control as opposed to just monitoring because the UX (or even software) isn't ideal it wants to provide more functionality in that respect.

Regardless I've cleaned up the code around these references so they are more consistent. If you really want them taken out, i'll have to refactor how it's been done.

Reply:
As this is also in library rather than just UI it makes much more sense to just have one ControlMode, I think two are too confusing. The UI doesn't matter _as much_ as it's unlikely to be used elsewhere.